src/midgard/tiles.cc

```diff
@@ -91,7 +102,7 @@ template <class coord_t> struct closest_first_generator_t {
       corners.emplace_back(x1, seed.second);
     }
     for (const auto& c : corners) {
-      auto d = seed.Distance(c);
+      auto d = seed.DistanceSquared(c);
```

Comment:
I'm not familiar with the algorithm used here, but will it work even though we;re returning square of the distance instead of the distance? Also maybe rename the variable for clarity (if sq distance can be returned)?

Reply:
Yeah - I also updated the caller of this code to compare the squared distance, so it works out.

Reply:
the algorithm is just something home grown from me. iirc it just uses the distance as a comparison, the absolute values of those dont really matter just so that order-wise you have them working out the same is all that should be needed

Reply:
Thanks for the explanation @kevinkreiser @danpat . So as long as the values returned by the `binner` is not used directly, it should be fine.